### DISCUSSION TITLE
The unit of output normal

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#225] [Q&A]
Hello developers,
  I've tried to compute ZTD from GNSS PPP, aming to  get the post variance through inversion of the Q(which is the normal equations):
![image](https://github.com/groops-devs/groops/assets/59271400/7f4fdc27-4ac1-4153-8f37-0e14e9449cc1)
But I found the values of Q is so large that the inversion of it will be very small, which can reach 10-E6, so I wonder the unit of them. The unit of computed ZTD  is 'm', so does unit of Q is same as ZTD and reveal the precision of ZTD?
regards,
Yongzhao Fan

**Comment by tmayerguerr** [comment]
Hi Yongzhao Fan,

Yes, the units should be [m].
In our experience, the estimated accuracies are an order of magnitude too optimistic (i.e. the variance is too small by a factor of 10^2). We suspect that this is because we neglect the correlations in the observations.

regards
Torsten
